tools/generator-bot-template-generator/app/templates/generators/app/templates/emptyBot.botproj

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/BotFramework-Composer/main/Composer/packages/server/schemas/botproject.schema",
+  "name": "emptyBot",
```

Comment:
Should we be renaming these files from emptyBot? Or is that being handled elsewhere?

Reply:
That is being tracked here https://github.com/microsoft/botframework-components/issues/563

For R12, we are assuming creation within Composer and Composer handles the changing of end file names and this variable to the bot name the user chooses on creation.